pom.xml

```diff
@@ -1793,6 +1811,26 @@
                                 <compilerArgs combine.self="append">
                                     <compilerArg>-parameters</compilerArg>
                                 </compilerArgs>
+                                <testExcludes>
+                                    <testExclude>**/*NoParametersCompilerFlag*</testExclude>
+                                </testExcludes>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>noparameters-testCompile</id>
+                            <phase>test-compile</phase>
+                            <goals>
+                                <goal>testCompile</goal>
+                            </goals>
+                            <configuration>
+                                <skip>${maven.compiler.testSources.noParamterCompilation.skip}</skip>
+                                <fork>true</fork>
+                                <proc>none</proc>
+                                <executable>${java-version.test.compiler}</executable>
+                                <compilerArgs combine.self="override" />
```

Comment:
Is this really necessary?

As far as I can tell, `-parameters` is not in the parent configuration, so you don't need to override it.

And by overriding, you're removing some useful parameters, such as `-Xlint:unchecked`.

Reply:
hmm, I'll give it a try to see if all would work fine and push the change. The idea for overriding it here was to make 10000% sure that we see all parameters we are passing for this execution so that either now or in the future nothing will sneak in 😃